src/app/api/newspaper/[title_id]/route.ts

```diff
@@ -10,15 +10,14 @@ import {createCatalogMissingNewspaperDtoFromIssue} from '@/models/CatalogMissing
 interface IdParams { params: { title_id: string} }
 
 // GET api/newspaper/[title_id]?box=[box]
-export async function GET(req: NextRequest, params: IdParams): Promise<NextResponse> {
-  const box = req.nextUrl.searchParams.get('box');
-  if (!box) return NextResponse.json({error: 'Box not provided'}, {status: 400});
+export async function GET(req: NextRequest): Promise<NextResponse> {
+  const boxId = req.nextUrl.searchParams.get('box_id');
+  if (!boxId) return NextResponse.json({error: 'Box not provided'}, {status: 400});
 
   return await prisma.newspaper.findMany({
```

Comment:
Kan vell bruke findUnique her siden boxId er unique PK

Reply:
Denne er for å hente alle newspapers som har denne boxId'en så da må man vel hente many

Reply:
ahh så feil🤦